src/containers/results.js

```diff
@@ -1,13 +1,63 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import * as actions from '../store/actions/index'
+
+import Layout from '../misc/layout';
+import ResultItem from '../components/results/resultItem';
 
 class Results extends Component {
-  render (){
+  
+  componentWillUnmount(){
+    this.props.clearState();
+  }
+  
+  render() {
+    
+    let drinks = this.props.results.map((drink, key) => {
+      return (
```

Comment:
you can use implicit return for this, we are not using any logic inside this function